packages/material-ui/src/Fade/Fade.js

```diff
@@ -66,20 +66,22 @@ const Fade = React.forwardRef(function Fade(props, ref) {
   const handleEntering = normalizedTransitionCallback(onEntering);
 
   const handleEnter = normalizedTransitionCallback((node, isAppearing) => {
-    reflow(node); // So the animation always start from the start.
+    if (node) {
```

Comment:
This would avoid the git diff to be noisy

```suggestion
    if (!node) {
      return;
    } 
```

Reply:
> This would avoid the git diff to be noisy

Disable whitespace in the diff.

Reply:
What about the future self that will want to "git blame" 😁?

Reply:
May be we can wait to see if people are confused with this error or not.